build.gradle

```diff
@@ -78,9 +78,9 @@ dependencies {
     compileOnly 'com.google.code.findbugs:jsr305:3.0.1'
 
     compile 'org.apache-extras.beanshell:bsh:2.0b6'
-    compile 'com.beust:jcommander:1.66'
+    compile 'com.beust:jcommander:1.72'
 
-    compile 'org.apache.ant:ant:1.9.7'
+    compile 'org.apache.ant:ant:1.9.7', optional
```

Comment:
@krmahadevan , you might want to take a look at this: https://stackoverflow.com/questions/25345804/optional-gradle-dependencies-for-maven-libraries

Reply:
@missedone - Thanks for sharing that information Nick. I had a gut feel that perhaps the [publishing-maven](https://github.com/cbeust/testng/blob/master/gradle/publishing-maven.gradle) build file may need to be updated with this.